src/Microsoft.Data.SqlClient/src/Microsoft/Data/SqlClient/DataClassification/SensitivityClassification.cs

```diff
@@ -0,0 +1,94 @@
+        public string Name { get; private set; }
+
+        /// <include file='..\../../../../../../doc/snippets/Microsoft.Data.SqlClient.DataClassification/Label.xml' path='docs/members[@name="Label"]/Id/*' />
+        public string Id { get; private set; }
+
+        /// <include file='..\../../../../../../doc/snippets/Microsoft.Data.SqlClient.DataClassification/Label.xml' path='docs/members[@name="Label"]/ctor/*' />
+        public Label(string name, string id)
+        {
+            Name = name;
+            Id = id;
+        }
+    }
+
+    /// <include file='..\../../../../../../doc/snippets/Microsoft.Data.SqlClient.DataClassification/InformationType.xml' path='docs/members[@name="InformationType"]/InformationType/*' />
+    public class InformationType
+    {
+        /// <include file='..\../../../../../../doc/snippets/Microsoft.Data.SqlClient.DataClassification/InformationType.xml' path='docs/members[@name="InformationType"]/Name/*' />
+        public string Name { get; private set; }
+
+        /// <include file='..\../../../../../../doc/snippets/Microsoft.Data.SqlClient.DataClassification/InformationType.xml' path='docs/members[@name="InformationType"]/Id/*' />
+        public string Id { get; private set; }
+
+        /// <include file='..\../../../../../../doc/snippets/Microsoft.Data.SqlClient.DataClassification/InformationType.xml' path='docs/members[@name="InformationType"]/ctor/*' />
+        public InformationType(string name, string id)
+        {
+            Name = name;
+            Id = id;
+        }
+    }
+
+    /// <include file='..\../../../../../../doc/snippets/Microsoft.Data.SqlClient.DataClassification/SensitivityProperty.xml' path='docs/members[@name="SensitivityProperty"]/SensitivityProperty/*' />
+    public class SensitivityProperty
+    {
+        /// <include file='..\../../../../../../doc/snippets/Microsoft.Data.SqlClient.DataClassification/SensitivityProperty.xml' path='docs/members[@name="SensitivityProperty"]/Label/*' />
+        public Label Label { get; private set; }
+
+        /// <include file='..\../../../../../../doc/snippets/Microsoft.Data.SqlClient.DataClassification/SensitivityProperty.xml' path='docs/members[@name="SensitivityProperty"]/InformationType/*' />
+        public InformationType InformationType { get; private set; }
+
+        /// <include file='..\../../../../../../doc/snippets/Microsoft.Data.SqlClient.DataClassification/SensitivityProperty.xml' path='docs/members[@name="SensitivityProperty"]/ctor/*' />
+        public SensitivityProperty(Label label, InformationType informationType)
+        {
+            Label = label;
+            InformationType = informationType;
+        }
+    }
+
+    /// <include file='..\../../../../../../doc/snippets/Microsoft.Data.SqlClient.DataClassification/ColumnSensitivity.xml' path='docs/members[@name="ColumnSensitivity"]/ColumnSensitivity/*' />
```

Comment:
Same here. That happens in almost every file.

Reply:
I guess it failed to push. It's done!